packages/curly/curly.0.3.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "Curly is a brain dead wrapper around the curl command line utility"
+maintainer: ["rudi.grinberg@gmail.com"]
+authors: ["Rudi Grinberg"]
+license: "ISC"
+homepage: "https://github.com/rgrinberg/curly"
+bug-reports: "https://github.com/rgrinberg/curly/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.03.0"}
+  "base-unix"
+  "result"
+  "alcotest" {with-test}
+  "cohttp-lwt-unix" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
    "@runtest" {with-test & ocaml:version >= "4.08"}
```

Reply:
Let's skip tests unless >= 4.08

Reply:
was that because of this?
```
[ERROR] Sorry, resolution of the request timed out.
        Try to specify a more precise request, use a different solver, or increase the allowed time by setting OPAMSOLVERTIMEOUT to a bigger value (currently, it is set to 500.0 seconds).
```
If so this is a known opam bug I'm currently debugging (https://github.com/ocaml/opam/issues/5422) and this change won't really do anything about it (the solver is called before running the tests) and I'd rather have them around just in case

Reply:
there was an actual failure on 4.02 and 4.03. I can change that to 4.04+ (the next version will be 4.08+ anyway)